src/Layout/redist/targets/OverlaySdkOnLKG.targets

```diff
@@ -38,13 +41,22 @@
     <Copy SourceFiles="@(ToolsetToOverlay)"
           DestinationFiles="@(ToolsetToOverlay->'$(SdkOutputDirectory)\%(RecursiveDir)%(Filename)%(Extension)')" />
 
+     <!-- Copy dotnet-watch files -->
```

Comment:
There is no good way to test it, but try to make a change in dotnet-watch and exam it reflects in the output of `./build.cmd -pack`. Razor has an issue that the overlay did not really work, and it is still the stage0 over get used and published. And... it is very likely this way now. We assemble dotnet-watch in dotnet/install. So whatever get produced here is likely get overridden https://github.com/dotnet/installer/blob/861a1dd12cb80bd834d0e51442d46ee7d1a4023f/src/redist/targets/BundledDotnetTools.targets

Reply:
My plan was to continue publishing this as a NuGet package but change the installer to start picking up the package from this repo (instead of AspNetCore). Do you foresee issues with this approach?

Reply:
I prefer it not to, it is very unintuitive. It would surprise people when debugging. And no SDK component goes a separate route. The fact we have this separate nuget package would be completely due to historical reason.